### DISCUSSION TITLE
checkpoint started at high step rate???

### DISCUSSION THREAD

**Discussion by wktra** [d8ahazard/sd_dreambooth_extension#298] [Q&A]
I started a new training and the first checkpoint produced was named model_5700.ckpt, followed by model_6400.ckpt, and model_7100.ckpt. Did my settings accidentally cause this???  I'm confused. What happened to the earlier checkpoints?

<img width="728" alt="Screenshot 2022-11-22 025141" src="https://user-images.githubusercontent.com/117292904/203269058-4ae9abdd-1648-4a7d-a7fd-86c40f402825.png">

**Comment by d8ahazard** [comment]
The number of trained steps *should* be based off of the config file in the model directory, which *should* only be updated when the model is saved.

Perhaps something got messed up when you created the model, or you used the same name or something?
